### PR TITLE
update for fall 2024

### PR DESCRIPTION
**Pull request by ptbarnum4**
image

### CODE REVIEW

File: src/Forms/Form.HighschoolApplication.tsx [hunk 1]
```diff
@@ -108,14 +108,11 @@ const HighSchoolApplication = ({ onSubmitComplete }: HighSchoolApplicationProps)
...
-        errs[n] = getStepErrors(Number(n));
-        return errs;
-      },
-      {} as Record<number, string[]>,
-    );
+    const stepErrs = Object.keys(stepFields).reduce((errs, stepNum) => {
+      const n = Number(stepNum);
+      errs[n] = getStepErrors(Number(n));
+      return errs;
+    }, {} as Record<number, string[]>);
```
Comment (harveysanders) [inline src/Forms/Form.HighschoolApplication.tsx:115]: I wonder what prettier rule this is. You and I always seem to get this change

File: pages/programs/highschool/apply.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ const HighschoolSignup: NextPage<HighschoolSignupProps> = ({ courses }) => {
                 <div className='left-col'>
                   <div className='hs-program-overview'>
                     <p>
-                      {`This summer, we’re offering half day summer coding camps, with in-person and virtual options. You can enroll in one or the other- there is no hybrid option. If you're able to arrange for reliable transportation to our learning center in the Marigny, we recommend in-person classes. We have a great lab with brand new equipment and an awesome staff to get you started on your coding journey. You'll also meet other students from a wide range of schools and backgrounds!`}
+                      {`This fall, we’re offering half day fall coding camps, with in-person and virtual options. You can enroll in one or the other- there is no hybrid option. If you're able to arrange for reliable transportation to our learning center in the Marigny, we recommend in-person classes. We have a great lab with brand new equipment and an awesome staff to get you started on your coding journey. You'll also meet other students from a wide range of schools and backgrounds!`}
```
Comment (mraychaudhuri) [inline pages/programs/highschool/apply.tsx:110]:
```suggestion
                      {`This fall, we’re offering after-school courses with in-person and virtual options. You can enroll in one or the other- there is no hybrid option. If you're able to arrange for reliable transportation to our learning center in the Marigny, we recommend in-person classes. We have a great lab with brand new equipment and an awesome staff to get you started on your coding journey. You'll also meet other students from a wide range of schools and backgrounds!`}
```

File: src/Forms/Form.HighschoolApplication.tsx (hunk 1 above)
Comment (ptbarnum4) [reply src/Forms/Form.HighschoolApplication.tsx:115]: my guess would be the [printWidth](https://github.com/OperationSpark/operationspark-org-website-2022/blob/main/.prettierrc#L9)

Comment (ptbarnum4) [reply src/Forms/Form.HighschoolApplication.tsx:115]: I could set it back to 80 (default)

File: pages/programs/highschool/apply.tsx (hunk 2 above)
Comment (ptbarnum4) [reply pages/programs/highschool/apply.tsx]: updated https://github.com/OperationSpark/operationspark-org-website-2022/pull/166/commits/f3926a7d316d5309e8e41867da839a614d0f6e0c

File: pages/programs/highschool/apply.tsx [hunk 3]
```diff
@@ -55,20 +55,20 @@ const HighschoolSignup: NextPage<HighschoolSignupProps> = ({ courses }) => {
           >
             <div className='program-header'>
               <div className='header-card'>
-                <h1 className='dynamic-xl secondary'>Summer 2024</h1>
+                <h1 className='dynamic-xl secondary'>Fall 2024</h1>
                 <h2 className='dynamic-h2 secondary'>Coding Camp Application</h2>
                 <p className='dynamic-txt'>
                   Open to rising sophomores, juniors, seniors, and 2024 grads
```
Comment (mraychaudhuri) [inline pages/programs/highschool/apply.tsx:77]: Open to sophomores, juniors, and seniors (Graduating classes of 2025, 2026, and 2027).

File: pages/programs/highschool/apply.tsx [hunk 4]
```diff
@@ -224,7 +224,7 @@ const HighschoolSignup: NextPage<HighschoolSignupProps> = ({ courses }) => {
               <br />
               <p>
                 {`If you go to school in a neighboring parish outside New Orleans, we may be running a
```
Comment (mraychaudhuri) [inline pages/programs/highschool/apply.tsx:242]: Remove this entire sentence about neighboring parishes

**Review by mraychaudhuri — COMMENTED** [review]
See two comments

File: pages/programs/highschool/apply.tsx (hunk 3 above)
Comment (ptbarnum4) [reply pages/programs/highschool/apply.tsx:77]: fixed: https://github.com/OperationSpark/operationspark-org-website-2022/pull/167/commits/edbd1e4acdb0161b32268560ccdd9cbf393df7db

File: pages/programs/highschool/apply.tsx (hunk 4 above)
Comment (ptbarnum4) [reply pages/programs/highschool/apply.tsx:242]: removed: https://github.com/OperationSpark/operationspark-org-website-2022/pull/167/commits/20048e4e6550ba0eebfaa82c17d8662f9344e8cd

**Pull request merged**